tests/test_main.py

```diff
@@ -20,7 +20,7 @@ def tearDown(self):
         sys.path[:] = self.old_sys_path
 
     def test_run(self):
-        with patch("mesa.visualization.ModularServer") as ModularServer:
+        with patch("mesa.flat.visualization.ModularServer") as ModularServer:
```

Comment:
This solves the problem, but I still need to know the underlying cause. It seems there is a collision between https://github.com/projectmesa/mesa/blob/77599fa67e74331ec958482ff4d053bb4da514b6/mesa/__init__.py#L13 and the actual `mesa/visualization` folder https://github.com/projectmesa/mesa/tree/main/mesa/visualization.

Reply:
When an import happens.

Reply:
I renamed `mesa/visualization` to `mesa/viz`, and
```python
with patch("mesa.visualization.ModularServer") as ModularServer:
    ....
```
works.

Reply:
What's the way to go? leave it as it is now or change the name of the folder? I think changing the name would be a breaking change (?)

Reply:
Yeah, changing the folder name is a breaking change for 1 of the 3 supported way to do imports. I will go through the Python 3.11 changelog by the end of the weekend.